andela/phlox-ah
# #160451503 Add functionality to rate articles

#### What does this PR do?
This pull request adds the functionality to rate an article.

#### Description of Task to be completed?
- Write tests for rating articles
- Write the code to rate an article

#### How should this be manually tested?
After cloning the project, run `npm install`, then `npm run watch` to start the server. Using postman and as a signed in user, on sending an integer or float as 'rating' to the rate API endpoint along with the correct url of an article, then an article should be rated.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#160451503
#### Questions:```
None
